config/samples/keda_v1alpha1_cloudevents.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: keda.sh/v1alpha1
+kind: CloudEvent
+metadata:
+  name: cloudevents-sample
+  namespace: keda
+spec:
+  clusterName: cluster-sample
+  eventHandlers:
+  - type: cloud-event-http
+    name: cloud-event-http-sample
```

Comment:
I can't see where these fields are used in the code. I guess that the type was for choosing the handler and the name maybe for filtering.
Based on @tomkerkhove 's spec, we were going to create another CRD for the event sources and maybe this info should be there?

Reply:
Sorry that I made some rollbacks from the array-type design. There may be some omissions, which I will check once the final CloudEvent spec is decided.